### ISSUE TITLE
Metal Slug Attack Reloaded

### ISSUE THREAD

**Issue by legolas119** [Ryujinx/Ryujinx-Games-List#4996]
### Game Name

Metal Slug Attack Reloaded

### Game Update Version

1.0.1

### Emulator Build Version

1.1.1336

### Graphics Backend

OpenGL and Vulkan

### Hardware Specs

CPU: (AMD Ryzen 3700X)
GPU: (RTX 2060 Super)
GPU Driver version: (552.22)
RAM: (16GB)

### Game State

The game immediately crashes as soon as i start it with the emulator

### Log File

[Ryujinx_1.1.1336_2024-06-19_23-29-46.log](https://github.com/user-attachments/files/15906085/Ryujinx_1.1.1336_2024-06-19_23-29-46.log)

**Comment by antoniodesousa** [comment]
v1.0.1 is working fine for me.

Title ID 010073401B8DE000
Ryujinx 1.1.1336
Vulkan

**Comment by legolas119** [comment]
yeah, now it works for me too! i don't know why yesterday it still continue to crash... i have firmware 18.0.0 i think it is the newest one

thanks
